homeassistant/helpers/entity_registry.py

```diff
@@ -780,6 +790,11 @@ async def _async_migrate(
         for entity in data["entities"]:
             entity["id"] = uuid_util.random_uuid_hex()
 
+    if old_major_version < 2 and old_minor_version < 5:
```

Comment:
There never was a version 0.
```suggestion
    if old_major_version == 1 and old_minor_version < 5:
```